TestIs2WavAudioPlayer/Program.cs

```diff
@@ -0,0 +1,34 @@
+using System.Diagnostics;
+using System.Threading;
+using AudioPlayer;
+using nanoFramework.Hardware.Esp32;
+using nanoFramework.System.IO.FileSystem;
+
+// Beware: if you stop debugging while the file is open,
+// re-plugin the USB cable to the MCU will re-start the application
+// therefore might re-open the wav file again and also "lock" it again.
+// thus we explicitly wait until Debugger is attached, so that we can "reset"
+while (!Debugger.IsAttached) Thread.Sleep(500);
```

Comment:
explicit bracket please to avoid confusions. We're using it everywhere, including in samples.